department-of-veterans-affairs/va.gov-cms
# VACMS-4918: Updated toolbar permissions for various roles

- Updated various roles permissions in the drupal admin permissions page.
- Updated roles, added permmission in PHPUnit test again.

## Description

closes #4918

## Testing done
Passes all tests

## Screenshots
![VACMS-4918-updated-toolbar-permissions](https://user-images.githubusercontent.com/846871/152581334-22d645a7-2767-4c83-a9ab-3afbcbcad69c.png)


## QA steps

As user administrator
1. [ ] Go to `/admin/people/permissions`, scroll down to "Toolbar" to verify and confirm the "Use toolbar" permission is assigned to the "Authenticated user" (and all other "Content Creator *" roles)

As a user content creator
1. [ ] Verify and confirm that you can see the toolbar

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
